h2o-core/src/main/java/water/api/RegisterV3Api.java

```diff
@@ -464,6 +464,14 @@ public void registerEndPoints(RestApiContext context) {
     context.registerEndpoint("list_rest_capabilities",
             "GET /3/Capabilities/API", CapabilitiesHandler.class, "listRest",
             "List of all registered Rest API capabilities");
+
+    context.registerEndpoint("import_grid_search",
+            "POST /3/Grid/import", GridImportExportHandler.class, "importGrid",
+            "Import previously saved grid model");
+
+    context.registerEndpoint("export_grid_search",
+            "POST /3/Grid/export", GridImportExportHandler.class, "exportGrid",
+            "Export a Grid and its models.");
```

Comment:
also, for those building on top on the REST API, would be nice to align with models here as well:
```
    // Model serialization - import/export calls
    context.registerEndpoint("importModel",
            "POST /99/Models.bin/{model_id}", ModelsHandler.class, "importModel",
            "Import given binary model into H2O.");

    context.registerEndpoint("exportModel",
            "GET /99/Models.bin/{model_id}", ModelsHandler.class, "exportModel",
            "Export given model.");
```
I have difficulties to understand the GET for export, but for the rest we could use similar URLs.

Reply:
@michalkurka any plan to officialise those "experimental" `/99/` APIs to `/3/`?
Also asking because AutoML is also using `/99/`: we could duplicate the handlers for the lifetime of a major release, and switch all clients to use `/3/`.

Reply:
Introducing /99/ shouldn't have been done. However, if we move it - we will certainly break something for people who use portions of the API directly. There was a jira to move everything out of /99/ it but it is not something is a priority right now.

I think from AutoML point of view - I would just move it.

All new stuff should go to /3/.

Reply:
Using {grid_id} is definitely a good idea.

Reply:
In-url {grid_id} param for export done.

Reply:
@Pscheidl this is binary import, right?
I we specify this in the URL (like with `/Models.bin/` vs. `/Models.mojo/`), I think you'll have more flexibility in the future if for example we want to have a feature allowing to export all grid models as mojo at once: you'll then be able to use `/Grid.bin/{grid_id}/export` vs. `/Grid.mojos/{grid_id}/export` and possibly support future formats in a consistent way.
Just my suggestion.

Reply:
Yes, this looks reasonably good.